test/app/config/application.rb

```diff
@@ -33,5 +38,7 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    config.active_record.default_column_serializer = YAML
```

Comment:
This is the only line I have to change the configurations manually. Others are changed by rails update script automatically.